packages/compass-connections-navigation/src/base-navigation-item.tsx

```diff
@@ -114,12 +114,9 @@ export const NavigationBaseItem: React.FC<NavigationBaseItemProps> = ({
       <div className={cx('item-wrapper', itemWrapperStyles)} style={style}>
         {isExpandVisible && (
           <ExpandButton
-            onClick={(evt) => {
-              if (isExpandDisabled) return;
-              evt.stopPropagation();
```

Comment:
I think this is still desirable because what I noticed - once you connect, and click on the expanded caret to collapse the tree, the default action (to open the tab for databases list) happens instead of collapse.

Reply:
I was hoping you'd share context on this. I forgot to ask directly 😬
I'll add it back and write a comment on why it's there 👍